src/EventListeners.ts

```diff
@@ -686,7 +686,42 @@ TODO: Fix this entire tab it's utter shit
     DOMCacheGetOrSet(`${String(key)}`).addEventListener('click', (event) => player.blueberryUpgrades[`${String(key)}`].buyLevel(event))
   }
 
+  // BLUEBERRY LOADOUTS
+  for (let i = 1; i <= 8; i++) {
+    // eslint-disable-next-line
+    DOMCacheGetOrSet(`blueberryLoadout${i}`).addEventListener('mouseover', () => createLoadoutDescription(i, player.blueberryLoadouts[i] ?? { ambrosiaTutorial: 0 }))
+    // eslint-disable-next-line
+    DOMCacheGetOrSet(`blueberryLoadout${i}`).addEventListener('click', () => loadoutHandler(i, player.blueberryLoadouts[i] ?? { ambrosiaTutorial: 0 }))
+  }
```

Comment:
```suggestion
  const blueberryLoadouts = document.querySelectorAll('[id^="blueberryLoadout"]')
  
  for (const loadout of blueberryLoadouts) {
    // eslint-disable-next-line
    loadout.addEventListener('mouseover', () => createLoadoutDescription(i, player.blueberryLoadouts[i] ?? { ambrosiaTutorial: 0 }))
    // eslint-disable-next-line
   loadout.addEventListener('click', () => loadoutHandler(i, player.blueberryLoadouts[i] ?? { ambrosiaTutorial: 0 }))
  }
```

Also, the eslint-disable comments should be removed.